pyproject.toml

```diff
@@ -188,3 +185,42 @@ version = { attr = "nemo_curator.package_info.__version__" }
 [tool.setuptools.packages.find]
 include = ["*"]
 exclude = ["tests", "tests.*"]
+
+[tool.ruff]
+line-length = 119
```

Comment:
Should line length be 119 or 88 like it was before?

Reply:
I changed it to follow what we have in https://github.com/NVIDIA/NeMo/blob/4609f97f476e66d8c833bf5aa9134b4d827c9da5/pyproject.toml#L86 based on @ayushdg 's comment